brave/brave-core
# Adds table refresh

Resolves https://github.com/brave/brave-browser/issues/1652

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

- enable rewards
- close rewards settings page
- visits some sites (10s+)
- open rewards settings page
- make sure that table is populated

## Reviewer Checklist:
